README.md

```diff
@@ -151,6 +151,8 @@ app.set('view engine', 'jade');
 
 ## Plugins
 
+You can find all of the PostHTML plugins in the [searchable catalog](http://maltsev.github.io/posthtml-plugins/).
```

Comment:
Maybe `almost all` instead of `all` ? ;-) Just you can't guarantee that you collect all of them.